### DISCUSSION TITLE
'addKtlintCheckGitPreCommitHook' is not appeared even plugin 'org.jlleitschuh.gradle.ktlint-idea' is applied.

### DISCUSSION THREAD

**Discussion by Xinkle** [JLLeitschuh/ktlint-gradle#487] [Q&A]
Hello, I'm using kitlint-gradle version 9.4.1

when i apply plugin `org.jlleitschuh.gradle.ktlint-idea`, task `ktlintApplyToIdea`, `ktlintApplyToIdeaGlobally` are showed but `addKtlintCheckGitPreCommitHook`, `addKtlintFormatGitPreCommitHook` are not showed.

above features are supported only in version `10.0.0`?
(i cannot use version 10.0.0 cause of internal network issue(related in gradle version))

**Comment by Tapchicoma** [comment]
Interesting, according to the code - this tasks are always added when plugin is applied to the root project. Is it possible to provide a sample that reproduces the issue?

**Reply by Xinkle** [reply]
@Tapchicoma
Currently i using script on root gradle file like below
`        classpath "org.jlleitschuh.gradle:ktlint-gradle:10.1.0"`
`    apply plugin: "org.jlleitschuh.gradle.ktlint" // Version should be inherited from parent`

but even I using version 10.1.0, helper tasks(like `ktlintApplyToIdea`) not shows on gradle script. (main functions (like `ktLintCheck`) are working)
Need I write it on the issue tab?
